### PR TITLE
[Player-4060] Seek buttons visibility in live video

### PR DESCRIPTION
**Pull request by SZadorozhnyiEpam**
Performed changes:
For DVR:
If scrubber in a live region then fast forward button will have grey color and not react to any touches.
For Live without DVR support:
Not yet implemented since we don't have a clear indicator of how to detect if Live has DVR or not.
But since we have `forceDvrDisabled` in skin.config and if this value is set to true then seek buttons will be invisible for the user.

### CODE REVIEW

File: sdk/react/panels/videoView.js [hunk 1]
```diff
@@ -272,6 +272,8 @@ class VideoView extends React.Component {
 
   _renderPlayPause = (show) => {
     const iconFontSize = ResponsiveDesignManager.makeResponsiveMultiplier(this.props.width, UI_SIZES.VIDEOVIEW_PLAYPAUSE);
+    const seekVisible = !this.props.config.live.forceDvrDisabled;
+    const notInLiveRegion = this.props.playhead <= this.props.duration * 0.95;
```
Comment (AlinaVoronkovaEpam) [inline sdk/react/panels/videoView.js]: what does 0.95 mean? Could you move it to constants?

**Review by AlinaVoronkovaEpam — COMMENTED** [review]
LGTM. Please make minor changes. See my comment.

File: sdk/react/panels/videoView.js (hunk 1 above)
Comment (SZadorozhnyiEpam) [reply sdk/react/panels/videoView.js]: 0.95 is a value from which player thinks that Live video should display Live icon, so we also disable FF button to synchronize them both.

>Could you move it to constants? 
Will do

**Review by akshaydch11 — APPROVED** [review]
LGTM

File: sdk/react/panels/videoView.js (hunk 1 above)
Comment (brol1dev) [reply sdk/react/panels/videoView.js]: agree with @AlinaVoronkovaEpam , wherever you found the usage of 0.95 please move it all to a constant so that it's easier to understand.

File: sdk/react/widgets/VideoViewPlayPause.js [hunk 2]
```diff
@@ -144,8 +147,8 @@ class VideoViewPlayPause extends React.Component {
   render() {
     const seekButtonScale = 0.5;
     const playPauseButton = this._renderPlayPauseButton();
-    const backwardButton = this._renderSeekButton(BACKWARD, seekButtonScale);
-    const forwardButton = this._renderSeekButton(FORWARD, seekButtonScale);
+    const backwardButton = this._renderSeekButton(BACKWARD, seekButtonScale, true);
```
Comment (brol1dev) [inline sdk/react/widgets/VideoViewPlayPause.js:156]: When forceDvrDisabled is on, both seek buttons should be hidden because you shouldn't be able to scrub back in time. Add the logic for this button too.

**Review by brol1dev — CHANGES_REQUESTED** [review]
Overall it looks good, but I added a couple of comments that I believe are important.

File: sdk/react/widgets/VideoViewPlayPause.js (hunk 2 above)
Comment (brol1dev) [reply sdk/react/widgets/VideoViewPlayPause.js:156]: my bad, realized the logic is correct here. Nothing to do. Sorry @SZadorozhnyiEpam

**Review by brol1dev — APPROVED** [review]
LGTM. Just added the comment about the `0.95` constant. I'll try to add the change on my side so we can create a new RC.

Thanks

File: sdk/react/widgets/VideoViewPlayPause.js [hunk 3]
```diff
@@ -112,15 +114,15 @@ class VideoViewPlayPause extends React.Component {
...
+  _renderSeekButton = (name, iconScale, active) => {
+    if (!this.props.showSeekButtons || !this.props.seekEnabled) {
       return <View/>
     }
     const fontStyle = {fontSize: this.props.fontSize * iconScale, fontFamily: this.props.icons[name].fontFamily};
     const sizeStyle = {width: this.props.buttonWidth, height: this.props.buttonHeight};
     const opacity = {opacity: this.state.skipButtons.animationOpacity};
     const animate = {transform: [{scale: this.state.skipButtons.animationScale}]};
-    const buttonColor = {color: !!this.props.buttonColor ? this.props.buttonColor : "white"};
+    const buttonColor = {color: active ? !!this.props.buttonColor ? this.props.buttonColor : "white" : "gray"};
```
Comment (pilievOoyala) [inline sdk/react/widgets/VideoViewPlayPause.js]: Please don't nest turnary statements. They are hard to follow. Can you separate out this logic into if statements and then assign buttonColor the result.

Comment (brol1dev) [reply sdk/react/widgets/VideoViewPlayPause.js]: changed it

**Comment by brol1dev** [comment]
@pilievOoyala, @akshaydch11 please review again. Added a couple of commits

**Comment by akshaydch11** [comment]
Looks good now

**Comment by pilievOoyala** [comment]
LGTM
